app.js

```diff
@@ -2,6 +2,9 @@
 const express = require('express');
 const app = express();
 const mongoose = require('mongoose');
+const booking = require('./models/booking.js');
+const user = require('./models/user.js');
+const note = require('./models/user.js');
```

Comment:
1. You don't need `.js` extension when requiring js files.
2. Models are more like classes so use PascalCase rather then camelCase
2. Let's make a module out of all database related stuff. So, make a `db` folder (or just rename this `models` one). Put all your model files there. Make `index.js` file where you will require all modules and then export them under one object. Something like this:
```js
const Booking = require('./models/booking.js');
...
module.exports = { 
  Booking,
  ...
};
```
Then you treat it like a single module and, in your `app.js`, just require it 
```js
require('./db');
```
Also now you have all db stuff under one module and if you need a single model e.g. in routes you can always destructure:
```js
const { Booking } = require('./db');
```
Later on, when we are done with models, we'll add more db stuff under this module, like mongo connection etc...